java/Oleksii-Riabov/src/com/zoolatech/lecture4/tasks/_2/Tree.java

```diff
@@ -0,0 +1,50 @@
+package com.zoolatech.lecture4.tasks._2;
+
+public class Tree {
+
+    class Node {
```

Comment:
Does the class Node need to have an access to some inner state of the class Tree? Do we need to make it as an inner class?

Reply:
Class Node is moved out of Tree class.

Reply:
Inner class is a class defined in other class, but it's not static. The class should be kept in the Tree class, but simply defined as static.

Reply:
Make Node static class of Tree class.

Reply:
it seems you haven't pushed your changes

Reply:
Yes, sorry. Already pushed my changes.